src/Microsoft.Health.Fhir.Core/Features/Operations/Export/CreateExportRequestHandler.cs

```diff
@@ -76,11 +76,45 @@ public async Task<CreateExportResponse> Handle(CreateExportRequest request, Canc
             // Otherwise, we will create a new export job. This will be a best effort since the likelihood of this happen should be small.
             ExportJobOutcome outcome = await _fhirOperationDataStore.GetExportJobByHashAsync(hash, cancellationToken);
 
+            ExportJobFormatConfiguration formatConfiguration = null;
+
+            if (request.Format != null)
+            {
+                formatConfiguration = _exportJobConfiguration.ExportJobFormats.FirstOrDefault(
+                (ExportJobFormatConfiguration formatConfig) => formatConfig.Name.Equals(request.Format, StringComparison.OrdinalIgnoreCase));
+            }
+
+            if (formatConfiguration == null)
+            {
+                if (request.ContainerName != null)
```

Comment:
>request.ContainerName != null [](start = 20, length = 29)

I think you can push this condition inside FirstOrDefault and that would make code a bit less bloated.
`formatConfiguration = _exportJobConfiguration.ExportJobFormats.FirstOrDefault((ExportJobFormatConfiguration formatConfig) => request.ContainerName!=null? formatConfig.ContainerDefault:formatConfig.NoContainerDefault);`